ladybug/src/main/java/org/frankframework/ibistesttool/MessageEncoder.java

```diff
@@ -25,16 +25,28 @@
 import org.apache.commons.io.input.BoundedInputStream;
 import org.apache.commons.io.input.BoundedReader;
 import org.frankframework.stream.Message;
+import org.springframework.beans.factory.InitializingBean;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import lombok.SneakyThrows;
 import nl.nn.testtool.Checkpoint;
 import nl.nn.testtool.MessageEncoderImpl;
 
-public class MessageEncoder extends MessageEncoderImpl {
+public class MessageEncoder extends MessageEncoderImpl implements InitializingBean {
 
 	private @Autowired int maxMessageLength;
 
+	// Both Ladybug and the present class in the Frank!Framework
+	// truncate messages as configured through maxMessageLength.
+	// In this class we keep one character more such that
+	// Ladybug can detect when a message is being truncated.
```

Comment:
It would be nice to not duplicate the comment in two classes. A possible solution would be to add something like the following with the comment to one of the classes and use this var in both classes:

protected static final long MAX_MESSAGE_LENGTH_ADJUSTMENT = 1L;